src/store/notaryKey.ts

```diff
@@ -0,0 +1,45 @@
+
+
+export enum ActionType {
+  SetKey = 'notaryKey/setKey'
+}
+
+export const setKey = (key: string) => ({
+  type: ActionType.SetKey,
+  payload: key
+})
+
+
+export type Action<payload = any> = {
+  type: ActionType;
+  payload: payload;
+  error?: boolean;
+  meta?: any;
+};
+
+
+type State = {
+  key: string;
+}
+
+const initState: State = {
+  key: `-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEBv36FI4ZFszJa0DQFJ3wWCXvVLFr\ncRzMG5kaTeHGoSzDu6cFqx3uEWYpFGo6C0EOUgf+mEgbktLrXocv5yHzKg==\n-----END PUBLIC KEY-----`
```

Comment:
I'd prefer to avoid duplicate copies in the source files.
Maybe we can move the public keys to a json file that we use here and in src/components/Header/index.tsx ?

Reply:
oh yeah, will change 👍